packages/components/organisms/f-cookie-banner/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v2.2.0
```

Comment:
Think you may need to rebase against master, as it's pulling in some extra commits from f-cookie-banner?